src/validation/ValidationSchema.ts

```diff
@@ -2,51 +2,37 @@ import * as yup from 'yup';
 import AmountType from '../types/AmountType';
 
 const validationSchema = yup.object({
-  name: yup.string().required('Please enter the scholarship name'),
-  deadline: yup
-    .date()
-    .required('Please enter the scholarship deadline')
-    .typeError('Please enter a valid date'),
-  description: yup
-    .string()
-    .required('Please enter the scholarship description'),
-  website: yup
-    .string()
-    .url('Website must be a valid URL')
-    .required('Please enter the scholarship website'),
+  name: yup.string().required('name'),
+  deadline: yup.date().required('deadline').typeError('validDate'),
```

Comment:
Rather than having the translation keys here (the i18next-parser will not pick it up), how about making this a function that takes in the Translation object itself? E.g.
```ts
const validationSchema = (t) => ({
  name: yup.string().required(t('name')),
  deadline: yup.date().required(t('deadline')).typeError(t('validDate'))
  ...
})
```

Reply:
hmm doesnt seem to work for all keys. For example, `validAmount` stays as is and doesn't get translated